pkg/component/controller/cplb/tcpproxy/tcpproxy.go

```diff
@@ -1,24 +1,8 @@
-// Copyright 2017 Google Inc.
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-//
-// Modifications made by Mirantis Inc., 2024.
-// Copyright 2017 Google Inc.
-//
-// Copyright 2024 Mirantis, Inc.
```

Comment:
I think this wasn't true. The copyright claim should have been "k0s authors" here.